core/src/main/java/org/mule/component/BindingInvocationHandler.java

```diff
@@ -75,7 +74,9 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
         MuleMessage message = createMuleMessage(args);
 
         // Some transports such as Axis, RMI and EJB can use the method information
-        message.setInvocationProperty(MuleProperties.MULE_METHOD_PROPERTY, method.getName());
+
+        // TODO MULE-9449 Remove Java Component Bindings
+        //message.setInvocationProperty(MuleProperties.MULE_METHOD_PROPERTY, method.getName());
```

Comment:
Looks like the binding won't work anymore after this change, so why to keep this commented?